### PR TITLE
{bio}[goolf/1.4.10] MACS 1.4.2-1 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 2
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/ca423f294b3f0f3d5562 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/aa4dbed58726c0794cc0 for a full test report.

File: easybuild/easyconfigs/m/MACS/MACS-1.4.2-1-goolf-1.4.10-Python-2.7.5.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = "PythonPackage"
+
+name = 'MACS'
+version = '1.4.2-1'
```
Comment (boegel) [inline easybuild/easyconfigs/m/MACS/MACS-1.4.2-1-goolf-1.4.10-Python-2.7.5.eb]: no download available for 1.4.2-1, only for 1.4.2?

Comment (boegel) [inline easybuild/easyconfigs/m/MACS/MACS-1.4.2-1-goolf-1.4.10-Python-2.7.5.eb]: Ah, wait, it is available at https://github.com/downloads/taoliu/MACS/MACS-1.4.2-1.tar.gz

Comment (boegel) [inline easybuild/easyconfigs/m/MACS/MACS-1.4.2-1-goolf-1.4.10-Python-2.7.5.eb]: But you can't download directly from that URL, it seems like you need to use https://cloud.github.com/downloads/taoliu/MACS/MACS-1.4.2-1.tar.gz instead...

Comment (boegel) [inline easybuild/easyconfigs/m/MACS/MACS-1.4.2-1-goolf-1.4.10-Python-2.7.5.eb]: Well, no, that doesn't work either (SSLv3 handshake error). :(

Comment (boegel) [inline easybuild/easyconfigs/m/MACS/MACS-1.4.2-1-goolf-1.4.10-Python-2.7.5.eb]:
```
== 2015-08-15 10:09:12,543 main.filetools WARNING IOError occurred while trying to download https://cloud.github.com/downloads/taoliu/MACS/MACS-1.4.2-1.tar.gz to /user/data/gent/vsc400/vsc40023/EasyBuild/sources/m/MACS/MACS-1.4.2-1.tar.gz: <urlopen error [Errno 1] _ssl.c:492: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
```

**Comment by boegel** [comment]
merged via #1869

**Pull request merged**